data_interrogator/interrogators.py

```diff
@@ -191,9 +186,13 @@ def is_excluded_model(self, model_class):
         app_label = model_class._meta.app_label
         model_name = model_class._meta.model_name
 
-        if self.allowed == Allowable.ALL_MODELS:
-            return False
-        return app_label in self.excluded or (app_label, model_name) in self.excluded
+        if app_label in self.excluded or (app_label, model_name) in self.excluded:
+            return True
+        else:
+            if self.allowed == Allowable.ALL_MODELS:
+                return False
+            else:
+                return app_label in self.allowed or (app_label, model_name) in self.allowed
```

Comment:
Pretty minor style thing, but we could drop the else's here to cut down on code / indentation. Thoughts?

Reply:
Done